packages/docs/src/stories/templates/teaser-media.stories.ts

```diff
@@ -142,7 +138,10 @@ export const TeaserMediaWithCopyright = {
       <sd-teaser-media variant="gradient-white" class="max-w-[600px]">
         <img slot="media" src="./placeholders/images/architecture.jpg" class="aspect-video object-cover" alt="" />
         <h3 slot="headline">Headline Media Teaser</h3>
-        <div class="flex flex-col sd-copyright gradient-white" style="--copyright: '© Union Investment 2024'">
+        <div
+          class="flex flex-col sd-copyright sd-copyright--color-black gradient-white"
+          style="--copyright: '© Union Investment 2024'"
```

Comment:
This should be wrapped around image like in the examples in the docs?

Reply:
To be honest I'm not sure what is the best approach. From what I can see this is the easier and cleaner way. If wrapping the image with a div, wich is a slot, or adding the copyright to the teaser media (as the image wrapper), additional CSS is required to fix the styles.

Reply:
I see – the issue is coming from the fact, that the image is in the background and that the copyright needs it's own space, right?

I better understand your approach now. Looks valid, let's go for it.